arangod/RestServer/arangod.cpp

```diff
@@ -151,141 +152,62 @@ static int runServer(int argc, char** argv, ArangoGlobalContext& context) {
-    server.addFeature<DatabaseFeature>();
-    server.addFeature<DatabasePathFeature>();
-    server.addFeature<EndpointFeature, HttpEndpointProvider>();
-    server.addFeature<EngineSelectorFeature>();
-    server.addFeature<EnvironmentFeature>();
-#ifdef TRI_HAVE_GETRLIMIT
-    server.addFeature<FileDescriptorsFeature>();
-#endif
-    server.addFeature<FlushFeature>();
-    server.addFeature<FortuneFeature>();
-    server.addFeature<FoxxFeature>();
-    server.addFeature<FrontendFeature>();
-    server.addFeature<GeneralServerFeature>();
-    server.addFeature<GreetingsFeature>();
-    server.addFeature<InitDatabaseFeature>(nonServerFeatures);
-    server.addFeature<LanguageCheckFeature>();
-    server.addFeature<LanguageFeature>();
-    server.addFeature<TimeZoneFeature>();
-    server.addFeature<LockfileFeature>();
-    server.addFeature<LogBufferFeature>();
-    server.addFeature<LoggerFeature>(true);
-    server.addFeature<MaintenanceFeature>();
-    server.addFeature<MaxMapCountFeature>();
-    server.addFeature<NetworkFeature>();
-    server.addFeature<NonceFeature>();
-    server.addFeature<PrivilegeFeature>();
-    server.addFeature<QueryRegistryFeature>();
-    server.addFeature<RandomFeature>();
-    server.addFeature<ReplicationFeature>();
-    server.addFeature<ReplicatedLogFeature>();
-    server.addFeature<ReplicationMetricsFeature>();
-    server.addFeature<ReplicationTimeoutFeature>();
-    server.addFeature<SchedulerFeature>();
-    server.addFeature<ScriptFeature>(&ret);
-    server.addFeature<ServerFeature>(&ret);
-    server.addFeature<ServerIdFeature>();
-    server.addFeature<ServerSecurityFeature>();
-    server.addFeature<ShardingFeature>();
-    server.addFeature<SharedPRNGFeature>();
-    server.addFeature<ShellColorsFeature>();
-    server.addFeature<ShutdownFeature>(
-        std::vector<std::type_index>{std::type_index(typeid(ScriptFeature))});
-    server.addFeature<SoftShutdownFeature>();
-    server.addFeature<SslFeature>();
-    server.addFeature<StatisticsFeature>();
-    server.addFeature<StorageEngineFeature>();
-    server.addFeature<SystemDatabaseFeature>();
-    server.addFeature<TempFeature>(name);
-    server.addFeature<TtlFeature>();
-    server.addFeature<UpgradeFeature>(&ret, nonServerFeatures);
-    server.addFeature<V8DealerFeature>();
-    server.addFeature<V8PlatformFeature>();
-    server.addFeature<V8SecurityFeature>();
-    server.addFeature<transaction::ManagerFeature>();
-    server.addFeature<VersionFeature>();
-    server.addFeature<ViewTypesFeature>();
-    server.addFeature<aql::AqlFunctionFeature>();
-    server.addFeature<aql::OptimizerRulesFeature>();
-    server.addFeature<pregel::PregelFeature>();
-
-#ifdef ARANGODB_HAVE_FORK
-    server.addFeature<DaemonFeature>();
-    server.addFeature<SupervisorFeature>();
-#endif
-
-#ifdef _WIN32
-    server.addFeature<WindowsServiceFeature>();
-#endif
-
-#ifdef USE_ENTERPRISE
-    setupServerEE(server);
```

Comment:
Not sure if this setup call is now somewhere or just lost. Am not yet through the full PR yet.